structuralcodes/codes/mc2010/_concrete_creep_and_shrinkage.py

```diff
@@ -148,671 +183,537 @@ def _check_env_temp(T: float) -> None:
             (temperature corrected) age of the concrete when loaded,
             as calculated by Eq. 5.1-71b.
 
-    returns:
+    Returns:
         numpy.ndarray: Multiplcation factor beta_dc_t for the
             considered values of time.
     """
-    return ((time - t0) / (beta_h + (time - t0))) ** gamma_t0
+    return ((time - t0) / (_beta_h + (time - t0))) ** _gamma_t0
 
 
-def calc_drying_creep_coefficient(
-    beta_dc_fcm: float,
-    beta_dc_RH: float,
-    beta_dc_t0: float,
-    beta_dc_t: np.ndarray,
+def phi_dc(
+    _beta_dc_fcm: float,
+    _beta_dc_RH: float,
+    _beta_dc_t0: float,
+    _beta_dc_t: np.ndarray,
 ) -> np.ndarray:
     """Calculate drying creep coefficient.
 
     Defined in fib Model Code 2010 (2013), Eq. 5.1-67.
 
-    args:
-        beta_dc_fcm (float): Multiplication factor that accounts for
+    Args:
+        _beta_dc_fcm (float): Multiplication factor that accounts for
             the effect of the strength of the concrete, as calculated
             by Eq. 5.1-68.
-        beta_dc_RH (float): Multiplication factor that accounts for the
+        _beta_dc_RH (float): Multiplication factor that accounts for the
             effect of the relative humidity of the environment, as
             calculated by Eq. 5.1-69.
-        beta_dc_t0 (float): multiplication factor that accounts for
+        _beta_dc_t0 (float): multiplication factor that accounts for
             the effect of the (temperature corrected) age of the
             concrete when loading is applied, as calculated by Eq.
             5.1-70.
-        beta_dc_t (numpy.ndarray): multiplication factor that accounts
+        _beta_dc_t (numpy.ndarray): multiplication factor that accounts
             for the different considered values of time, as calculated
             by Eq. 5.1-71a.
 
-    returns:
+    Returns:
         numpy.ndarray: Drying creep coeffcient.
     """
-    return beta_dc_fcm * beta_dc_RH * beta_dc_t0 * beta_dc_t
+    return _beta_dc_fcm * _beta_dc_RH * _beta_dc_t0 * _beta_dc_t
 
 
-def _calc_k_sigma(sigma: float, fcm: float) -> float:
+def k_sigma(sigma: float, fcm: float) -> float:
     """Calculate the ratio between the applied stress and the mean
         concrete compressive strength.
 
     Defined in fib Model Code 2010 (2013), Eq. 5.1-74.
 
-    args:
+    Args:
         sigma (float): The compressive stress applied to the concrete
             at ts in MPa.
         fcm (float): The mean compressive strength of the concrete in
             MPa.
 
-    returns:
+    Returns:
         float: Absolute value of the ratio between the stress in the
             concrete and the mean concrete strength.
     """
+    _check_initial_stress(sigma, fcm)
     return abs(sigma / fcm)
 
 
-def calc_creep_coefficient(
-    time: np.ndarray,
-    t0: float,
-    T_cur: float,
-    fcm: float,
-    rh: float,
-    notional_size: float,
-    sigma: float,
-    agg_type: str,
-    cem_class: str,
+def phi(
```

Comment:
I don't know here if the user is expected that this function is 5.1-63 it is strange to provide sigma and fcm. Those are used for effect of high stresses in 5.1-74.
I would split the phi function for 5.1-63 and phi_sigma for 5.1-74.
The signatures would then become:
`def phi(_phi_bc, _phi_dc)`
and
`def phi_sigma(phi, sima, fcm)`

I have not strong feelings in doing so, but if we want to leave as it is it should be updated the docstrings highlighting that this function is implementation of5%.1-63 + 5.1-74?

Reply:
Good comment. I think we can always refactor later if it is needed, but the docstring should always be clear. @slobbeat, I suggest to keep the function as is and just adjust the docstring 😃

Reply:
Done